nemo/collections/tts/modules/glow_tts_submodules.py

```diff
@@ -0,0 +1,643 @@
+# MIT License
```

Comment:
Can we follow this: https://github.com/NVIDIA/NeMo/pull/988#discussion_r465936184 for copyright headers?